docs/website/docs/dlt-ecosystem/file-formats/parquet.md

```diff
@@ -76,3 +77,18 @@ You can generate parquet files without timezone adjustment information in two wa
 2. Set the **timestamp_timezone** to empty string (ie. `DATA_WRITER__TIMESTAMP_TIMEZONE=""`) to generate logical type without UTC adjustment.
 
 To our best knowledge, arrow will convert your timezone aware DateTime(s) to UTC and store them in parquet without timezone information.
+
+
+### Row group size
+The `pyarrow` parquet writer writes each item, i.e. table or record batch, in a separate row group. 
+This may lead to many small row groups which may not be optimal for certain query engines. For example, `duckdb` parallelizes on a row group.
+`dlt` allows controlling the size of the row group by
+buffering and concatenating tables and batches before they are written. The concatenation is done as a zero-copy to save memory.
+You can control the memory needed by setting the count of records to be buffered as follows:
+```toml
+[extract.data_writer]
+buffer_max_items=10e6
+```
+Mind that `dlt` holds the tables in memory. Thus, 1,000,000 rows in the example above may consume a significant amount of RAM.
+
+`row_group_size` has limited utility with `pyarrow` writer. It will split large tables into many groups if set below item buffer size.
```

Comment:
Got it now, thanks! Maybe we can give a recommendation like this:
```suggestion
Setting `row_group_size` has limited utility with the `pyarrow` parquet writer because large source tables can end up fragmented into too many groups.
Thus, we recommend setting `row_group_size < buffer_max_items` only when the write_disposition is `"replace"`.
For all other write dispositions, we recommend the default `row_group_size` to avoid fragmentation.
```

Reply:
row groups do not map to write dispositions like this... I think this is only relevant to advanced users that optimize their parquet files for a particular query engine...